electricitymap/contrib/capacity_parsers/FI.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime, timedelta
+from typing import Any
+
+from requests import Response, Session
+
+from electricitymap.contrib.capacity_parsers.ENTSOE import (
+    fetch_production_capacity as fetch_entsoe_production_capacity,
+)
+from electricitymap.contrib.config import ZoneKey
+
+API_KEY = "a411d83c96bb4abb8812dba80bdaed64"
```

Comment:
We really should not store the API key here. It should be a ENV variable.

(PS: you should revoke this API key if possible as it's already in the git history).

Reply:
done!